### ISSUE TITLE
False negative for IssetArgumentExistenceInspection on array elements

### ISSUE THREAD

**Issue by fraserredmond** [kalessil/phpinspectionsea#995]
| Subject    | Details                                                  |
| :--------- | :------------------------------------------------------- |
| Issue type | Bug/False-Negative     |
| IDE        | PhpStorm v2018.2.1                          |
| Plugin     | Php Inspections (EA Extended), v3.0.5.1 |

Description or screenshot:
--
IssetArgumentExistenceInspection doesn't trigger for array variables, here's how it looks in IDE:

![image](https://user-images.githubusercontent.com/1796533/44269964-06a75a80-a204-11e8-9574-d166edaf8df6.png)

Reproduce with this code:

```
<?php

function example_falseNegative () {
    // Inspection does not show a warning, even though $foo isn't defined in the context.
    if (!empty($foo['bar'])) {
        echo '';
    }
}

function example_correct () {
    // Inspection shows a warning, correctly.
    if (!empty($foo)) {
        echo '';
    }
}

```

**Comment by kalessil** [comment]
This case is getting detected by Php Inspections (EA Ultimate), hence closing the issue.
